BLSQ/iaso
# IA-1822 remove the users many to many field from account model

Remove the field "users" on Account object. This field was not used and was causing confusion.

Related JIRA tickets : IA-1822

## Self proof reading checklist


## Changes

Remove the field "users" on Account object. This field was not used and was causing confusion.
Modified some fixtures
Modified some code used for tests

All python tests are passing.
I also searched for references to `.users` and `account.users` in all python files.
I manually opened the dashboard/frontend and checked all pages from the menu (not all functionalities though)
I checked the Django admin account page.

## How to test

⚠️  For reviewers, if you know any django management command that might use Account, please try it and check it locally, I didn't really test those and I don't think they are much covered by tests

Also if you know any page in the dashboard that might be broken or was using account.users (Normally nothing was using it but still)

Also @bmonjoie is there anything in the mobile app using the many to many relationship `Account.users` ?
